airbyte-integrations/connectors/source-google-analytics-v4/source_google_analytics_v4/spec.json

```diff
@@ -81,7 +81,9 @@
         "type": "string",
         "title": "Replication Start Date",
         "description": "The date in the format YYYY-MM-DD. Any data before this date will not be replicated.",
-        "examples": ["2020-06-01"]
+        "examples": ["2020-06-01"],
+        "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}$|^$|[\\s\\S]+$",
+        "format": "date"
```

Comment:
Do we have an old configuration validation?

Reply:
by default, dates that are valid for this source are in YYYY-MM-DD format, this change only adds validation to this exact configuration